core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -331,9 +353,12 @@ private static Deadline min(@Nullable Deadline deadline0, @Nullable Deadline dea
 
   @Override
   public void request(int numMessages) {
-    Preconditions.checkState(stream != null, "Not started");
```

Comment:
Why remove this line? `start()` still sets `stream` to non-`null`. I've got the same confusion elsewhere.